cmd/odf/ceph/rbd.go

```diff
@@ -21,6 +22,15 @@ var RbdCmd = &cobra.Command{
 		if err != nil {
 			logging.Fatal(err)
 		}
+		// If the user is trying to run "rbd ls" (without possible extra arguments)
+		if args[0] == "ls" && len(args) == 1 {
+			if len(args) > 1 {
+				logging.Warning("Ignoring extra arguments for 'rbd ls'; running only 'rbd ls'.")
+			}
+			// Call the vendor ListImages function to do the listing
+			rbd.ListImages(cmd.Context(), root.ClientSets, root.OperatorNamespace, root.StorageClusterNamespace)
+			return
+		}
```

Comment:
```suggestion
		// If the user is trying to run "rbd ls" (without possible extra arguments)
		if args[0] == "ls" && len(args) == 1 {
			if len(args) > 1 {
				logging.Warning("Ignoring extra arguments for 'rbd ls'; running only 'rbd ls'.")
			}
			// Call the vendor ListImages function to do the listing
			rbd.ListImages(cmd.Context(), root.ClientSets, root.OperatorNamespace, root.StorageClusterNamespace)
			return
		}
```

instead of this can we do similar to https://github.com/rook/kubectl-rook-ceph/pull/333/files#diff-9b7d33b239e9bb3a80d5cda418273877bb1398a5a0f928fc478ef98b37901c5dR44-R55 ?

Reply:
@subhamkrai I changed the code based on the upstream `[kubectl-rook-ceph](https://github.com/rook/kubectl-rook-ceph)' project, but I didn't add an option to list all images for a specific pool only `rbd ls`. Do you want me to `add rbd ls <pool name>`?